semantics/executable-spec/src/Data/Coders.hs

```diff
@@ -66,6 +65,10 @@ module Data.Coders
     dualCBOR,
     roundTrip,
     roundTrip',
+    embedTrip,
+    roundTripAnn,
+    embedTripAnn,
+    RoundTripResult,
```

Comment:
One thought,  the discussion of visual inspection of Duality refers to roundTrip as a way of proving (well testing) Duality. I though it might be nice if they were in the same file. But I see your point. I moved them to the generators file, as they would be useful for any Encoding-Decoder pair.